_activities/gravity.md

```diff
@@ -4,12 +4,22 @@ layout: activity
 # Content
 #
 title: "Gravity"
-title_fr: ""
+title_fr: "Gravité"
 age: 8-12
-objective: ""
-objective_fr: ""
-summary: ""
-summary_fr: ""
+objective: "Students will learn how to quantitatively describe how things fall under the affect of gravity. "
```

Comment:
We aren't really having them quantitatively describe how things fall under gravity. Let's change this to: "Students will learn how an object's mass, starting height, and initial velocity affect how it falls under the force of gravity."